src/OSPSuite.SimModel/src/Species.cpp

```diff
@@ -448,9 +448,14 @@ void Species::SetODEIndex(int newIndex)
 	m_ODEIndex = newIndex;
 }
 
-void Species::AppendUsedParameters(std::set<int> & usedParameterIDs) {
+void Species::AppendUsedParameters(std::set<int> & usedParameterIDs, bool includeInitialFormula) 
```

Comment:
@hdiedam Not sure why initial formula is skipped when getting used parameters for CPP-Export.
Thus I added optional parameter includeInitialFormula (false per default, so CPP-Export is not affected)

Reply:
The cppExport uses that function primarily to distinguish between "local" parameters (switched or somehow depending on y and t) and "global" parameters (depending only on free parameters) that can be precomputed. If they are depending only on some initial value y(t_0) and not on the state trajectory y(t) they still can be considered global.